homeassistant/components/rfxtrx/__init__.py

```diff
@@ -365,6 +381,13 @@ def name(self):
         """Return the name of the device if any."""
         return self._name
 
+    @property
+    def device_state_attributes(self):
+        """Return the device state attributes."""
+        if not self._event:
+            return None
+        return {ATTR_EVENT: "".join(f"{x:02x}" for x in self._event.data)}
```

Comment:
It's not so nice that we have to add this state attribute to restore state, if that's the only use case.

Reply:
Well it's the only thing that always contains enough data. The fact that data attributes are displayed in Lovelace is almost a bug. At least since I got a pull rejected to display it for other entity types than switch and sensor.

Reply:
We do this sometimes so I think it's ok, but it's a sign that something in our API isn't optimal.